Lib/defconQt/glyphView.py

```diff
@@ -187,27 +292,38 @@ def __init__(self, glyph, parent=None):
         glyphMenu.addAction("&Jump", self.changeGlyph, "J")
         menuBar.addMenu(glyphMenu)
 
-        toolBar = QToolBar(self)
+
+        self._displaySettings = DisplayStyleSettings("&Display", self, self._displayChanged)
+        menuBar.addMenu(self._displaySettings.menuWidget)
+
+        self._toolBar = toolBar = QToolBar(self)
         toolBar.setMovable(False)
         toolBar.setContentsMargins(2, 0, 2, 0)
-        selectionToolButton = toolBar.addAction("Selection", self.view.setSceneSelection)
+        selectionToolButton = toolBar.addAction("Selection", self._redirect('view', 'setSceneSelection'))
         selectionToolButton.setCheckable(True)
         selectionToolButton.setChecked(True)
         selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
-        penToolButton = toolBar.addAction("Pen", self.view.setSceneDrawing)
+        penToolButton = toolBar.addAction("Pen", self._redirect('view', 'setSceneDrawing'))
         penToolButton.setCheckable(True)
-        penToolButton.setIcon(QIcon(":/resources/curve.svg"))
-        rulerToolButton = toolBar.addAction("Ruler", self.view.setSceneRuler)
+        selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
+        rulerToolButton = toolBar.addAction("Ruler", self._redirect('view', 'setSceneRuler'))
         rulerToolButton.setCheckable(True)
-        rulerToolButton.setIcon(QIcon(":/resources/ruler.svg"))
-        knifeToolButton = toolBar.addAction("Knife", self.view.setSceneKnife)
+        selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
+        knifeToolButton = toolBar.addAction("Knife", self._redirect('view', 'setSceneKnife'))
         knifeToolButton.setCheckable(True)
-        knifeToolButton.setIcon(QIcon(":/resources/cut.svg"))
+        selectionToolButton.setIcon(QIcon(":/resources/cursor.svg"))
         # http://www.setnode.com/blog/right-aligning-a-button-in-a-qtoolbar/
         spacer = QWidget()
         spacer.setSizePolicy(QSizePolicy.Expanding, QSizePolicy.Expanding)
         toolBar.addWidget(spacer)
-        toolBar.addWidget(self.view.currentLayerBox())
+        self._currentLayerBox = QComboBox(self)
```

Comment:
Is the goal to have all the external UI elements in the MainWindow rather than in the View? I've found handling that in the View more convenient in practise.